docs/users.md

```diff
@@ -46,7 +46,7 @@ Credentials for these users are stored as a [Kubernetes Secrets](https://kuberne
 The Operator requires Kubernetes Secrets before Percona XtraDB Cluster is
 started. It will either use existing Secrets or create a new Secrets object with
 randomly generated passwords if it didn’t exist.
-The name of the required Secrets (`cluster1` by default)
+The name of the required Secrets (`cluster1-secrets` by default)
```

Comment:
I think it would be better just `Secret` since it is one `Secret` object with multiple passwords inside